README.md

```diff
@@ -1,35 +1,41 @@
 # Vivarium Core
 
-Vivarium Core provides a process interface and simulation engine for
-composing and executing integrative multi-scale models.
+[![Read the Docs](https://img.shields.io/readthedocs/vivarium-core)](https://vivarium-core.readthedocs.io/en/latest/)
+[![PyPI](https://img.shields.io/pypi/v/vivarium-core)](https://pypi.org/project/vivarium-core/)
+![GitHub branch checks state](https://img.shields.io/github/checks-status/vivarium-collective/vivarium-core/master)
```

Comment:
I think this will make the badge just an image. Can we make it a link too like the other ones?

Also, when I preview this file the badge says "pending" even though the tests are passing right now

Reply:
@U8NWXD -- where should this link to? https://github.com/vivarium-collective/vivarium-core/actions?

Reply:
How about https://github.com/vivarium-collective/vivarium-core/actions?query=branch%3Amaster so that we only get the runs on master?

Reply:
fixed!